cfn-resources/project-ip-access-list/resource-role.yaml

```diff
@@ -15,6 +15,13 @@ Resources:
             Principal:
               Service: resources.cloudformation.amazonaws.com
             Action: sts:AssumeRole
+            Condition:
```

Comment:
why are we adding it? didn't it work before without this change?

Reply:
I am not sure 🤔 this was added automatically by cfn generate. I will remove it

Reply:
removed